src/lib/chat/matrix-client.ts

```diff
@@ -455,6 +455,8 @@ export class MatrixClient implements IChatClient {
           event_id: parentMessage.messageId,
         },
       };
+      content['format'] = 'org.matrix.custom.html';
+      content['formatted_body'] = message;
```

Comment:
I don't think we should be doing this. It's not required and I don't think it's a good idea for the sending client to be telling the receiving client how to render things. That's just coupling things together for no reason.